### ISSUE TITLE
 An other suggestion: wlsunset (redshift equivalent)

### ISSUE THREAD

**Issue by taschenlampe** [openSUSE/openSUSEway#42]
Just an other suggestion :-) 
I use sway on my laptop and I really prefer a blue filter while looking at its screen at nights -> There is this nice little tool that adjusts the gamma for wayland/swaywm [wlsunset](https://sr.ht/~kennylevinsen/wlsunset/). Though there is no rpm for opensuse yet, but it's pretty easy to install.

**Comment by m-rey** [comment]
I've started packaging it here: https://build.opensuse.org/package/show/home:mrey/wlsunset

**Comment by jubalh** [comment]
Accepted SR https://build.opensuse.org/request/show/889691 to devel project: https://build.opensuse.org/package/show/X11:Wayland/wlsunset

**Comment by denisok** [comment]
cool, interesting how to integrate it to the solution ?

**Comment by denisok** [comment]
run it with something like:
`curl -s http://ip-api.com/json/$(curl -s https://ipinfo.io/ip) | jq '.lat,.lon' | xargs bash -c 'wlsunset -l $0 -L $1'` from sway config ? but what is the way to toggle on/off with way bar ? Something like `pkill wlsunset` ?

**Comment by jubalh** [comment]
I'm fairly certain that people will complain about making connections to some website without them being aware :D But I don't see another solution out of the box.

**Comment by m-rey** [comment]
wlsunset is packaged and in Factory now https://build.opensuse.org/package/show/openSUSE:Factory/wlsunset

**Comment by denisok** [comment]
> I'm fairly certain that people will complain about making connections to some website without them being aware :D But I don't see another solution out of the box.

I agree :) we could just add it as config file for sway with everything commented out, and ppl could uncomment if they wish to use it.

**Comment by etam** [comment]
There is GeoClue for getting location: https://gitlab.freedesktop.org/geoclue/geoclue/-/wikis/home

Found it used here: https://phabricator.kde.org/source/plasma-redshift-control/browse/master/package/contents/ui/config/ConfigAdvanced.qml

**Comment by denisok** [comment]
@krishjainx sure, please do. I don't know though that it should be enabled by default, not everyone would like that feature. But having it as an option - cool

**Comment by kskarthik** [comment]
this could be a nice addition. Can be added to zypper `Recommends`
